### PR TITLE
VER-80: some basic layout + functionality for mobile version

### PR DESCRIPTION
**Pull request by n3op2**
# What
Please refer to the screenshot, bus basically a skeleton for some flows and etc. Mostly used default components. Tests will be included as another PR. All properties are required appart from expiry date

- visit by localhost:3000/issue and recommended at least 320px width + 500px height
- using html 5 forms that allows to use require and other default properties, basica validation

## screenshots
Screenshot 2023-08-23 at 10 14 22
Screenshot 2023-08-23 at 10 13 55
Screenshot 2023-08-23 at 10 13 51
Screenshot 2023-08-23 at 10 11 51
Screenshot 2023-08-23 at 10 11 45
Screenshot 2023-08-23 at 10 11 36

### CODE REVIEW

File: src/pages/Issue/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      checks: data.checks ? JSON.parse(data.checks) : undefined,
+    }))
+    setStage((prev) => prev + 1)
+  }
+
+  if (fetching) return <Spinner />
+
+  return (
+    <Content>
+      {stage === 0 && (
```
Comment (n3op2) [inline src/pages/Issue/index.js:42]: rename to something sensible and maybe more clever mechanism

File: src/pages/Issue/Summary.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+        <Row>
+          <Button
+            onClick={(e) => {
+              e.preventDefault()
+              setStage((prev) => prev - 1)
+            }}
+          >
+            {'<'}
+          </Button>
+          <Button onClick={handleSubmit}>{'Confirm >'}</Button>
```
Comment (jonmattgray) [inline src/pages/Issue/Summary.js:148]: I get the following error when hitting confirm `FormData constructor: Argument 1 does not implement interface HTMLFormElement.`

File: src/pages/Issue/Summary.js [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  font-style: italic;
+  font-size: 8px;
+`
+
+const Td = styled('td')`
+  font-size: 8px;
+  align-text: center;
+`
+
+export default function Summary({ inputs, setStage, handleSubmit }) {
```
Comment (jonmattgray) [inline src/pages/Issue/Summary.js:37]: I know this is a basic layout for mobile but this summary is really small at my laptop resolution?
image

File: src/pages/Issue/CompanyDetailsForm.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        <Input
+          type="text"
+          required
+          name="name"
+          value={inputs.name}
+          placeholder="Company name"
+        />
+        <Input
+          required
+          type="number"
```
[the remark this answers is not in the document]
Comment (n3op2) [reply src/pages/Issue/CompanyDetailsForm.js:26]: maybe? In figma was just numbers

File: src/pages/Issue/Summary.js (hunk 2 above)
Comment (n3op2) [reply src/pages/Issue/Summary.js:148]: yea, confirm did not do anything, now it should just spin forever

File: src/pages/Issue/Summary.js (hunk 3 above)
Comment (n3op2) [reply src/pages/Issue/Summary.js:37]: updated

File: src/pages/Issue/Summary.js (hunk 2 above)
Comment (jonmattgray) [reply src/pages/Issue/Summary.js]: 👍  didn't realise it was a placeholder

File: src/pages/Issue/Summary.js (hunk 3 above)
Comment (jonmattgray) [reply src/pages/Issue/Summary.js:37]: yep looks better thanks

File: src/pages/Issue/ChecksForm.js [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+            <Grid.Panel area="issuance">
+              <DatePicker
+                type="date"
+                required
+                name="issuance"
+                onChange={(e) => {
+                  e.preventDefault()
+                  setSelected((prev) => [
+                    ...prev.filter((item) => item.value != el.value),
+                    { ...el, expiry: e.target.value },
```
Comment (jonmattgray) [inline src/pages/Issue/ChecksForm.js:72]: it looks like issuance and expiry are being set the wrong way round. This is the `issuance` picker but is setting the `expiry` value, and vice versa for the other date picker

Comment (n3op2) [reply src/pages/Issue/ChecksForm.js]: thanks, leftovers from troubleshooting multiple items

**Review by jonmattgray — APPROVED** [review]

**Pull request merged**